client/src/components/overview/Cart.jsx

```diff
@@ -101,13 +115,13 @@ class Cart extends React.Component {
     const skusList = Object.keys(skus);
     console.log('Cart sku #s', skusList);
 
-    const sizes = [];
+    const sizesList = [];
     skusList.forEach(sku => {
```

Comment:
I think this would be equivalent: `const sizesList = skusList.map(sku => skus[sku].size);`